README.md

```diff
@@ -43,6 +43,10 @@ Following mechanisms may be used for DUT control:
 
 ![regression-architecture](https://cloud.3mdeb.com/index.php/s/KkERgGoniBtjfC4/preview)
 
+### Current osfv_cli purpose
```

Comment:
I'd call this OSFV architecture

Reply:
title updated: https://github.com/Dasharo/open-source-firmware-validation/pull/459/commits/ede9434d7fa2744836168f82ed751eae806ecb54